packages/host/app/resources/search.ts

```diff
@@ -47,7 +47,13 @@ export class Search extends Resource<Args> {
 
   modify(_positional: never[], named: Args['named']) {
     let { query, realms, isLive, doWhileRefreshing } = named;
-    this.realmsToSearch = realms ?? this.realmServer.availableRealmURLs;
+    if (query === undefined) {
+      return;
+    }
```

Comment:
I think its very possible that we build the query asynchronously. It is no good, sometimes to query a default value as it will cause blinking and an unnecessary query.